### DISCUSSION TITLE
airflow in cluster Nomad

### DISCUSSION THREAD

**Discussion by yaelkadosh** [apache/airflow#26908] [General] [answered]
Hi
Do you also support Nomad Cluster?
Is there an official Nomad Pack?

**Comment by potiuk** [comment, marked as answer]
There are user using Airlfow in Nomad, so it is possible, but we do not have nor plan to release nomad pack for it as a community. We release Helm Chart for K8S https://airflow.apache.org/docs/helm-chart/stable/index.html and it is released and tested by us. 

BTW. I also know some users of our migrated from Nomad to K8S recently :)

**Reply by yaelkadosh** [reply]
Thanks
My customer has now switched from Kubernetes to Nomad
